GPU/Directx9/TextureCacheDX9.cpp

```diff
@@ -1639,6 +1639,7 @@ TextureCacheDX9::TexCacheEntry::Status TextureCacheDX9::CheckAlpha(const u32 *pi
 	return (TexCacheEntry::Status)res;
 }
 
+// TODO: xoffset and yoffset are unused - bug?
```

Comment:
Hmm, they're fixed at 0 also.  Maybe the plan was to support updating a subarea?  Sounds hard, maybe we should just remove.

-[Unknown]